frontend/src/components/CardBase/CardBase.tsx

```diff
@@ -0,0 +1,56 @@
+import Grid from "@mui/material/Unstable_Grid2";
+import {Button, Card, CardActions, CardContent, Chip, Tooltip, Typography} from "@mui/material";
+import {Psychology} from "@mui/icons-material";
+import {Tags} from "../../enums/tagEnums";
+import {Link} from "react-router-dom";
+import React, {ReactNode} from "react";
+import {Tag} from "../../models/Tag";
+
+const CardBase : React.FC<{
+    index: number, item: any, tags: Tag[], handleTags: any, cardActions: ReactNode, cardName: ReactNode, summary: ReactNode
+}> = ({index, item, tags, handleTags, cardActions, cardName, summary}) => {
+
+
+    return (
+        <Grid xs={12} sm={6} lg={4} xl={3} key={index}
```

Comment:
If the component is called CardBase, it should only be about a Card item, not a Grid of Cards, maybe use CardBase in a ItemGridBase or something similar